learning/tour-of-beam/frontend/.firebaserc

```diff
@@ -0,0 +1,24 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
check if we can create it from scratch instead of modifying

Reply:
File will be created during grade task now, file removed from PR